hack/install-deps.sh

```diff
@@ -134,7 +140,36 @@ fi
 # Install containerd
 checkout_repo ${CONTAINERD_PKG} ${CONTAINERD_VERSION} ${CONTAINERD_REPO}
 cd ${GOPATH}/src/${CONTAINERD_PKG}
-make
+if ${COOK_CONTAINERD}; then
+  # Verify that vendor.conf is in sync with containerd before cook containerd,
+  # this is a hard requirement.
+  if ! ${ROOT}/hack/update-vendor.sh -only-verify; then
```

Comment:
If this only verify `vendor.conf`, I think this `if statement` can be remove.
May be we can use `cp -rTn ${ROOT}/vendor/ vendor/` [here](https://github.com/containerd/cri-containerd/pull/527/commits/794543065a238b1a6ce1c826b1fdf23db91c9651#diff-6951ee2c812d7e05017a2bfb50060ef5R156)

Reply:
The idea here is to report out of the out of sync condition..